Community/manage_records/__init__.py

```diff
@@ -0,0 +1,12 @@
+# Copyright 2017 BlueCat Networks. All rights reserved.
```

Comment:
Please change date to 2019. There is another instance of 2017 in manage_records.py

Reply:
@delmeherbert Updated to change to 2019